Examples/MagicWeather/MagicWeather.xcodeproj/project.pbxproj

```diff
@@ -61,7 +61,7 @@
 		5576344125A3C06E00C0486A /* Views */ = {
 			isa = PBXGroup;
 			children = (
-				5576344225A3C08400C0486A /* PackageCell.swift */,
+				5576344225A3C084.1.0486A /* PackageCell.swift */,
```

Comment:
This seems broken? 🤔

Reply:
lol I suppose `sed` must have caught that 400?? I'm not even sure how that happened, I'll revert

Reply:
fixed